game/scripts/npc/abilities/techies_land_mines.txt

```diff
@@ -61,12 +61,12 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "building_damage_pct"                             "30"
+        "building_damage_pct"                             "125" //oaa
       }
       "06"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "activation_delay"                                "1.75"
+        "activation_delay"                                "1.75 1.75 1.75 1.75 1.5 1.25"
```

Comment:
I didn't want to fully change the numbers of activation delay at earlier levels so I instead just scaled it at OAA levels since it seems magical techies falls off a cliff late game (many opt for the physical build by then).